archivy/config.py

```diff
@@ -1,10 +1,15 @@
 import os
 import appdirs
 
+# version of archivy configuration file
+VERSION = 1
+
 
 class Config(object):
     """Configuration object for the application"""
 
+    VERSION = VERSION
```

Comment:
Maybe we can directly define it as `VERSION = 1` here?

Reply:
Yes, agreed.